app/services/update_user.rb

```diff
@@ -33,7 +33,24 @@ def update_sam
     reckoner.delay.set!
   end
 
+  def parsed_user_params
+    user_params.each_with_object({ }) do |(key, value), hash|
+      converted_value = nil_not_empty_string(value)
+      hash[key] = converted_value
+    end
+  end
```

Comment:
I see. So for those String attributes that are not required, you prefer them to be stored in the database as nil as opposed to an empty string. If that's the case, would this work?
```ruby
def normalized_user_params
  user_params.delete_if { |_, v| v == '' }
end
```